frontend/src/components/ImagePicker.js

```diff
@@ -3,12 +3,13 @@ import ImageUpload from './ImageUpload';
 import _ from 'underscore';
 
 const REG_VALID_URL = /^(https?:\/\/)?([\da-z\.-]+)\.([a-z\.]{2,6})([\/\w \.-]*)*\/?$/;
-const REG_VALID_TWITTER_USERNAME = /^[a-zA-Z0-9_]{1,15}$/;
+const REG_VALID_TWITTER_USERNAME = /^@?([a-zA-Z0-9_]{1,15})$/;
```

Comment:
This is going to break both the frontend validator and the backend User model. Backend expects the string without an @ symbol. Allowing users to enter it in the field makes sense and ideally we'd fix this both front and backend. For now, easiest fix is likely to remove the @ symbol before passing to the form. What do you think?